UnitedIncome/components
# [APP-2996] - Update itemization widget to support edit tooltip

### Description
Updates itemization widget to support edit tooltip

### Testing Instructions
1. Check out the default story for the itemization widget

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-2996)
